common/plugins/org.jboss.tools.common.ui/src/org/jboss/tools/common/ui/databinding/HostNameValidator.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2012 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.common.ui.databinding;
+
+import java.util.regex.Pattern;
+
+import org.eclipse.core.databinding.validation.IValidator;
+import org.eclipse.core.databinding.validation.ValidationStatus;
+import org.eclipse.core.runtime.IStatus;
+import org.jboss.tools.openshift.common.core.utils.StringUtils;
+
+/**
+ * @author Andre Dietisheim
+ */
+public class HostNameValidator implements IValidator {
+
+	private static final Pattern urlPattern =
```

Comment:
Turns out the urlPattern is actually a host pattern, not an url pattern, thus not duplicate. Renaming the constant.